test/notebook/test_spec.js

```diff
@@ -15,36 +15,35 @@
 // This is an e2e test suite that runs notebooks in a browser, execute all their cells
 // and make sure no errors were thrown.
 // It reads a spec.json file specifying which notebooks to run, and optionally a list
-// of cell numbers of ignore, which can be useful if specific cells are flaky
+// of cell numbers to ignore, which can be useful if specific cells are flaky
 // or intentionally erroneous.
 
-const assert = require('assert');
 const selenium = require('selenium-webdriver'),
       until = selenium.until;
-const test = require('selenium-webdriver/testing');
 const fs = require('fs');
-
-var driver = null;
+require('jasmine2-custom-message');
 
 const notebooks_config = JSON.parse(fs.readFileSync('notebook/spec.json'))
 const timeOut = 60000;
 
-test.before(function() {
-  this.timeout(timeOut);
-  driver = new selenium.Builder()
+function createDriver() {
```

Comment:
I had trouble with this before, but I tried it again and now it works. So, Done.